### ISSUE TITLE
`children is missing in props validation`

### ISSUE THREAD

**Issue by dcousens** [feross/standard#152]
When using React,  `standard` always complains about `this.props.children` not being defined in `propTypes`,  even though it is a complex type that is given by React it self.

I personally don't even know what type it would even be,  I know it is (kind of) array-like.

Is there a way we can avoid having to set up the `propType` validation for this property?

**Comment by cesarandreu** [comment]
The correct generic type is node.

```js
{
  propTypes: {
    children: React.PropTypes.node
  }
}
```


If it can only be a single child, it's `element`.

**Comment by feross** [comment]
I recently extracted the react-specific configs into their own shareable config. See: https://github.com/feross/eslint-config-standard/issues/2

You guys should just change whatever options you want in [eslint-config-standard-react](https://github.com/feross/eslint-config-standard-react). I've added you both as collaborators and npm owners. I don't use React, so I have no context or opinion about these issues. If you think there's likely to be controversy, open an issue / PR and wait a few days before merging. Same as we do on this repo.

`standard` depends on exact versions of the configs, so whenever you publish a new version of the react config, we'll need to bump the version here.
